modular_skyrat/modules/medical/code/smartdarts.dm

```diff
@@ -60,6 +60,12 @@
 		to_chat(user, span_notice("The [container] is unable to fit inside of the [src]! Try using a <b>SmartDart</b> instead."))
 		return FALSE
 
+/obj/item/gun/syringe/smartdart/examine(mob/user)
+	. = ..()
+
+	for(var/obj/item/reagent_containers/syringe/dart in syringes)
```

Comment:
```suggestion
	for(var/obj/item/reagent_containers/syringe/dart as anything in syringes)
```

Reply:
what does `as anything` do?

Reply:
Skips the typecheck when iterating through the list of syringes. As there won't be anything _other_ than `.../syringe/dart` in there, it's unnecessary to check the type of each one.
So it's a performance gain. Very minor, but this is used all over the codebase and they add up.

Reply:
gotcha